packages/design-system-docs/src/pages/startup/components.md

```diff
@@ -55,9 +55,9 @@ ReactDOM.render(
 
 See [React component documentation]({{root}}/components/button/#components.button.react) for more react examples, guidance, and detailed prop documentation.
 
-<h2>Using CSS components</h2>
+## Using CSS components
 
-If your project doesn't use React, we provide a CSS-only version of our components. This includes styles for every component in the library, but you’ll be responsible for writing the correct markup and updating classes and DOM attributes in response to user events.
+If your project doesn't use React, we provide a HTML/CSS-only version of our components. This includes styles for every component in the library, but you’ll be responsible for writing the correct markup and updating classes and DOM attributes in response to user events.
 
 This example assumes you've already added [design system styles via CSS or Sass]({{root}}/startup/sass-and-css/) by including either the CSS or Sass.
```

Comment:
consider updating to:

> This example assumes you've already added [design system styles via CSS or Sass]({{root}}/startup/sass-and-css/).

Reply:
Isnt that the same text?

Reply:
Removing this from the end of the sentence because it seems redundant `by including either the CSS or Sass.`

Reply:
Isn't that the same text?

Reply:
Ah I didnt catch that thanks